api/app/Models/Article.php

```diff
@@ -87,6 +87,20 @@ protected static function boot()
             }
             return true;
         });
+
+        static::created(function (Article $article) {
+            $articleComment = (new ArticleComment)->setArticle($article);
+            $articleComment->newDiscussion();
```

Comment:
Would it make more sense to have an `ArticleDiscussion` model, so that a new comment doesn't have to be instantiated to generate a discussion? It's just architecture, but it kinda seems like a mix of concerns.